### ISSUE TITLE
Standalone

### ISSUE THREAD

**Issue by asmodyus** [DillonN/NiceHashMiner#35]
Sorry for the dumbquestion is this a standalone version or an upgrade. Reason Im asking is I cannot seem to get the dual Mining working here is the extra Launch Parameters. -dpool stratum+tcp://decred.usa.nicehash.com:3354 -dwal 1QKKCC29LUyAtprKsrLsMmmMuWKS.worker1  -mode 0 -dcoin dcr -dcri 95

**Comment by ChriscomIT** [comment]
This is a standalone software with integrated miners. It's a fork/upgrade from the original version. The miners integrated however are for the use/are configured for the use of the software/GUI. They are not meant to be run standalone via cmd command or start .bat file.

**Comment by asmodyus** [comment]
Yeah I am trying to get the Dual Miner working. I'm reading different ways of doing it for my 1080Ti's but none of them are working.

**Comment by ChriscomIT** [comment]
Cause the miners in this package are preconfigured to run with the software. Run it via the GUI and u can set it up easily. If you want standalone miners then you need to download them from the respective website/GitHub releases.

**Comment by asmodyus** [comment]
Okay correct but what I am trying to do is get it to dual mine with the claymore, From what I have read I go to settings, Devices/Algorithms click on the claymore it should automatically start dual mining. Other people are say I need to put a command in the extra Launch parameters which I also did. But either way nothing is working for me on that aspect of the software.
